pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1276,6 +1276,8 @@ func (b *changefeedResumer) resumeWithRetries(
 			}
 		}
 
+		flowErr = changefeedbase.MaybeAnnotate(flowErr)
+
 		// Terminate changefeed if needed.
 		if err := changefeedbase.AsTerminalError(ctx, jobExec.ExecCfg().LeaseManager, flowErr); err != nil {
 			log.Infof(ctx, "CHANGEFEED %d shutting down (cause: %v)", jobID, err)
```

Comment:
Actually, looking at this code, when we print "shut down error" -- it appears to me that this error will happen when
for example, jobs registry is shutting down -- it will cancel ctx and AsTerminalError propagates context cancelation.
So, it seems to me, that at the very least we should suppress shutdown message when ctx.Err != nil
Alternative, we could check for <-drainCh to see if the registry is draining.